--- pkgs-void.list	2021-03-14 09:27:42.795179658 -0400
+++ pkgs-void.list	2021-03-14 09:22:20.202107907 -0400
@@ -1,10 +1,11 @@
+#
 GraphicsMagick
 MultiMC
 R
 Signal-Desktop
 alsa-plugins-pulseaudio
 apache-fop
-ario
+ario  # GUI (GTK) MPD client
 aspell-en
 aspell-ru
 autoconf
@@ -30,7 +31,7 @@
 docker
 docker-compose
 dunst
-element-desktop
+element-desktop  # Matrix.org client, replaces Riot
 encfs
 extra-cmake-modules
 feh
@@ -69,7 +70,7 @@
 lm_sensors
 lvm2
 lxdm
-mDNSResponder
+mDNSResponder  # Bonjour
 make
 man-pages-devel
 mate
@@ -81,12 +82,12 @@
 mpd
 mpv
 mupdf
-nawk
+nawk  # The one, true implementation of AWK
 ncmpc
 ncmpcpp
 ncurses-devel
 neofetch
-net-tools
+net-tools  # ifconfig et al
 netbsd-wtf
 network-manager-applet
 nmap
@@ -98,9 +99,9 @@
 parallel
 patch
 pcmanfm
-perl-List-MoreUtils-XS
+perl-List-MoreUtils-XS  # Dependency of feedgnuplot, which isn't in Void repos.
 pfetch
-picom
+picom  # Replaces compton.
 pidgin
 pkg-config
 python3-devel
@@ -131,7 +132,7 @@
 tmux
 tree
 udisks2
-ueberzug
+ueberzug  # For drawing images on terminals. Something built from src needed it.
 unixodbc-devel
 unzip
 vim
@@ -155,7 +156,7 @@
 xrandr
 xscreensaver
 xtools
-xwininfo
+xwininfo  # Needed for picom-trans to work, but isn't a dep of picom.
 youtube-dl
 zathura
 zathura-djvu
