--- pkgs-void.list	2021-03-06 15:39:19.227168042 -0500
+++ pkgs-void.list	2021-03-06 15:37:14.082174938 -0500
@@ -1,3 +1,4 @@
+#
 GraphicsMagick
 R
 Signal-Desktop
@@ -91,9 +92,9 @@
 parallel
 patch
 pcmanfm
-perl-List-MoreUtils-XS
+perl-List-MoreUtils-XS  # Dependency of feedgnuplot, which isn't in Void repos.
 pfetch
-picom
+picom  # Replaces compton.
 pidgin
 pkg-config
 python3-devel
@@ -144,7 +145,7 @@
 xrandr
 xscreensaver
 xtools
-xwininfo
+xwininfo  # Needed for picom-trans to work, but isn't a dep of picom.
 youtube-dl
 zathura
 zathura-djvu
