--- pkgs-void.list	2021-03-11 05:14:06.217196387 -0500
+++ pkgs-void.list	2021-03-11 05:13:00.283192775 -0500
@@ -1,9 +1,10 @@
+#
 GraphicsMagick
 R
 Signal-Desktop
 alsa-plugins-pulseaudio
 apache-fop
-ario
+ario  # GUI (GTK) MPD client
 aspell-en
 aspell-ru
 autoconf
@@ -30,7 +31,7 @@
 docker
 docker-compose
 dunst
-element-desktop
+element-desktop  # Matrix.org client, replaces Riot
 encfs
 extra-cmake-modules
 feh
@@ -62,14 +63,14 @@
 libX11-devel
 libXft-devel
 libXinerama-devel
-libcrypto46
+libcrypto46  # Erlang didn't work without it
 libreoffice
 libsodium-devel
 libtool
 lm_sensors
 lvm2
 lxdm
-mDNSResponder
+mDNSResponder  # Bonjour
 make
 man-pages-devel
 mate
@@ -81,12 +82,12 @@
 mpd
 mpv
 mupdf
-nawk
+nawk  # The one, true implementation of AWK
 ncmpc
 ncmpcpp
 ncurses-devel
 neofetch
-net-tools
+net-tools  # ifconfig et al
 netbsd-wtf
 network-manager-applet
 nmap
@@ -98,9 +99,9 @@
 parallel
 patch
 pcmanfm
-perl-List-MoreUtils-XS
+perl-List-MoreUtils-XS  # Dependency of feedgnuplot, which isn't in Void repos.
 pfetch
-picom
+picom  # Replaces compton.
 pidgin
 pkg-config
 python3-devel
@@ -129,7 +130,7 @@
 tmux
 tree
 udisks2
-ueberzug
+ueberzug  # For drawing images on terminals. Something built from src needed it.
 unixodbc-devel
 vim
 vlc
@@ -151,7 +152,7 @@
 xrandr
 xscreensaver
 xtools
-xwininfo
+xwininfo  # Needed for picom-trans to work, but isn't a dep of picom.
 youtube-dl
 zathura
 zathura-djvu
